docker/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM julia:latest
```

Comment:
Better to pin a version in case someone finds this file in months to years in the future